helm/aws-load-balancer-controller/templates/servicemonitor.yaml

```diff
@@ -29,7 +25,19 @@ spec:
   endpoints:
     - port: metrics-server
       path: /metrics
-      {{- with .Values.serviceMonitor.interval }}
+      scheme: http
```

Comment:
just for my understanding, why we specify `http` explicitly here?

Reply:
I always set the `scheme` so it's very clear what the intent is even for someone who's starting at the `ServiceMonitor` and working up through the other resources. From an implementation point of view this is something where even though the default is correct we 100% care about the value, so we should explicitly set it.